control-plane-operator/controllers/hostedcontrolplane/etcd/params.go

```diff
@@ -10,60 +12,65 @@ import (
 )
 
 type EtcdParams struct {
-	ClusterVersion            string
-	EtcdOperatorImage         string
-	OwnerRef                  config.OwnerRef `json:"ownerRef"`
-	OperatorDeploymentConfig  config.DeploymentConfig
-	EtcdDeploymentConfig      config.DeploymentConfig
-	PersistentVolumeClaimSpec *corev1.PersistentVolumeClaimSpec `json:"persistentVolumeClaimSpec"`
+	EtcdOperatorImage string
+
+	OwnerRef         config.OwnerRef `json:"ownerRef"`
```

Comment:
more out of curiosity, but why do we have json tags on these, do we ever serialize them?

Reply:
@alvaroaleman at some point I thought I would. We really should get rid of them.